dotnet/samples/AgentSyntaxExamples/BaseTest.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Reflection;
+using Configuration;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel;
+using RepoUtils;
+using Xunit.Abstractions;
+
+namespace Examples;
+
+public abstract class BaseTest
+{
+    /// <summary>
+    /// Flag to force usage of OpenAI configuration if both <see cref="TestConfiguration.OpenAI"/>
+    /// and <see cref="TestConfiguration.AzureOpenAI"/> are defined.
+    /// If 'false', Azure takes precedence.
+    /// </summary>
+    protected virtual bool ForceOpenAI { get; } = false;
+
+    protected ITestOutputHelper Output { get; }
+
+    protected ILoggerFactory LoggerFactory { get; }
+
+    protected string GetApiKey()
+    {
+        if (string.IsNullOrEmpty(TestConfiguration.AzureOpenAI.Endpoint))
```

Comment:
Does this need to account for the `ForceOpenAI ` flag as well or is that handled somewhere else?
```suggestion
        if (string.IsNullOrEmpty(TestConfiguration.AzureOpenAI.Endpoint) || ForceOpenAI)
```

Reply:
Doh!  Indeed...both spots.